### ISSUE TITLE
No buffer space available

### ISSUE THREAD

**Issue by FPokerFace** [JmFoces/Rehtse#2]
My Rehtse threw this error:

failed to read from netfilter queue (0 bytes): No buffer space available

I guess it's because queries my server received too many queries, so how to set the buffer larger?

**Comment by JmFoces** [comment]
Are you using jumbo frames ?
Please, pull last commit and let me know if it works.

**Comment by FPokerFace** [comment]
No, The listened packets are no more than 1000 octets. I'm experimenting DNS implementations and sending queries to the targeted nameserver deployed with Rehtse to modify the DNS response.

I built the Rehtse with the last commit on Ubuntu 22.04 and it works well. Each response of the targeted nameserver is modified well, even the queries per second(qps) reaches 100, i.e. the response targeted nameserver send reach 100 qps, Rehtse works fine. However, when the rate reaches 500qps or 1000qps, Rehtse quickly throw the error and shutdown.

**Comment by JmFoces** [comment]
I'm busy and I won't be able to investigate this and fix it fix soon. 
For now, can you remove the return statement at NFQueue.cpp line 82 ? 
Let me know the results please.

**Comment by FPokerFace** [comment]
OK I will try
